lib/ansible/modules/network/netvisor/pn_vrouter_ospf.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/python
+# Copyright: (c) 2018, Pluribus Networks
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = """
+---
+module: pn_vrouter_ospf
+author: "Pluribus Networks (@rajaspachipulusu17)"
+version_added: "2.9"
+short_description: CLI command to add/remove vrouter-ospf
+description:
+  - This module can be used to add OSPF protocol to vRouter
+    and remove OSPF protocol from a vRouter
+options:
+  pn_cliswitch:
```

Comment:
@rajaspachipulusu17 as mentioned in my earlier comments in other pluribus PRs, you have to make 1/some params as mandatory i.e. `required: true`, else user can just run the module without passing any param to the module.

Reply:
@justjais i think user cant execute without passing arguments, because i have used required_if condition, is that good enough?

Reply:
@rajaspachipulusu17 ok, in this case u can mention `['pn_vrouter_name', 'pn_network']` params as `required: true`, as for the state option respective params are required to be passed by user.

Reply:
@justjais i have made the change which you suggested :)